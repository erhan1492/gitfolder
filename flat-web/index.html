<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>example for photoshop</title>

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,300i,400" rel="stylesheet">
    
     <link href="css/font-awesome.min.css" rel="stylesheet">
    
    <link href="css/main.css" rel="stylesheet" >
    <link href="css/animate.min.css" rel="stylesheet"/>
    <link href="css/lightbox.css" rel="stylesheet">
   

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>
  <body>
    
    <header>
    	
    	<div class="container">
    		
    		    <div class="row">
    		    	
    		        	<a class="logo" href="#"><img src="img/logo.png" alt="erhan"/></a>
    		    	    <nav class="menu" id="menu">
    		    	    	
    		    	    	   <ul>
    		    	    	   	 
    		    	    	   	 <li><a href="#mainContent">HOME</a></li>
    		    	    	   	<li><a href="#services">SERVICES</a></li>
    		    	    	   	<li><a href="#works">WORKS</a></li>
    		    	    	   	<li><a href="#team">TEAM</a></li>
    		    	    	   	<li><a href="#contact">CONTACT</a></li>
    		    	    	   </ul>
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    	
    		    	    </nav>
    		    </div>
    		
    		
    		
    		
    		
    	</div>
    	
    	
    	
    </header> <!-- header burad bitti -->
    
    <section id="mainContent" class="text-center">
    	
    	
    	   <div class="container">
    	   	
    	   	      <div class="mainArea">
    	   	      	
    	   	      	    <h1>Modern Free Psd Template</h1>
    	   	      	   <p>Lorem ipsum dolor sit amet, numquam appareat liberavisse his ei, at eleifend praesent suavitate eam, nihil impetus apeirian ut est.  nec sonet prompta. Quando assentior interesset duo ex, ea soluta patrioque nam. Audire admodum vivendum ei his, ut esse velit reformidans eum.</p>
    	   	      	
    	   	      	
    	   	      	<button type="button" class="btn btn-success">SCROLL</button>
    	   	      	
    	   	      </div>
    	   	
    	   	
    	   	
    	   	
    	   	
    	   	
    	   	
    	   	
    	   </div>
    	
    	
    	
    	
    	
    </section> <!-- section burda bitiyor-->
    
    <section id="services" class="text-center">
    	
    	
    	<div class="conteiner">
    		
    		
    		
    		 <div class="servicestop">
    		 	
    		 	<h1>SERVICES</h1>
    		 	<p>Lorem ipsum dolor sit amet, numquam appareat liberavisse his ei, at eleifend praesent suavitate eam, nihil impetus apeirian ut est.<br> An nec sonet prompta. Quando assentior interesset duo ex, ea soluta 

patrioque nam. Audire admodum vivendum ei his, ut esse velit reformidans eum. Eos mazim choro ex, ancillae officiis salutandi pri an.
</p>


    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 </div>
    		
    		
    		<div class="servicesContent">
    			
    			<div class="row">
    				
    				   <div class="col-md-4 wow fadeIn" data-wow-delay="0.2s">
    				   	<i class="fa fa-desktop fa-4x" aria-hidden="true"></i>
    				   	<h2>Web Desigin</h2>
    				   	<p>In a professional context it often happens that private or corporate clients corder                        a publication </p>
    				   </div>
    				  
    				  <div class="col-md-4 wow fadeIn" data-wow-delay="0.4s">
    				   	<i class="fa fa-print fa-4x" aria-hidden="true"></i>
    				   	<h2>Print Desing</h2>
    				   	<p>In a professional context it often happens that private or corporate clients corder                        a publication </p>
    				   </div>
    				  
    				  <div class="col-md-4 wow fadeIn" data-wow-delay="0.8s">
    				   	<i class="fa fa-mobile fa-4x" aria-hidden="true"></i>
    				   	<h2>Mobile</h2>
    				   	<p>In a professional context it often happens that private or corporate clients corder                        a publication </p>
    				   </div>
    				  
    				
    				
    				
    			</div>
    			
    			
    			
    			
    			
    			
    			
    			
    		</div>
    		
    		
    		
    	</div>
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    </section>  <! -- section burda bitiyor-->




<!-- works baslangic -->



 
    <section id="works" class="text-center">
    	
    	
    	<div class="conteiner">
    		
    		
    		
    		 <div class="workstop">
    		 	
    		 	<h1>WORKS</h1>
    		 	<p>Lorem ipsum dolor sit amet, numquam appareat liberavisse his ei, at eleifend praesent suavitate eam, nihil impetus apeirian ut est.<br> An nec sonet prompta. Quando assentior interesset duo ex, ea soluta 

patrioque nam. Audire admodum vivendum ei his, ut esse velit reformidans eum. Eos mazim choro ex, ancillae officiis salutandi pri an.
</p>


    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 </div>
    		
    		
    		<div class="worksContent">
    			
    			<div class="row">
    				
    				<div class="col-md-3">
    				
    				<a href="img/work1.jpg" data-lightbox="work1" data-title="My caption">
    					<img  src="img/work1.jpg"/>
    					
    				</a>
    					
    					
    					
    				</div>
    				
    				<div class="col-md-3">
    					
    					
    						
    				<a href="img/work2.jpg" data-lightbox="work1" data-title="My caption">
    					<img  src="img/work2.jpg"/>
    					
    				</a>
    					
    					
    					
    				</div>
    				
    				<div class="col-md-3">
    						
    				<a href="img/works3.jpg" data-lightbox="work1" data-title="My caption">
    					<img src="img/works3.jpg"/>
    					
    				</a>
    					
    					
    					
    				</div>
    				
    				
    				<div class="col-md-3">
    					
    					
    						
    				<a href="img/works4.jpg" data-lightbox="work1" data-title="My caption">
    					<img src="img/works4.jpg"/>
    					
    				</a>
    					
    					
    					
    				</div>
    				
    				
    				
    				
    				
    				
    				
    				
    			</div>
    			
    			
    			
    			
    			
    			
    			
    		</div>
    		
    		
    		
    	</div>
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    </section>  <! -- section burda bitiyor-->



<! -- counter-->

	<section id="counter" class="text-center">
		
		
		
		<div class="container">
			
			<div class="row wow fadeInLeft">
				
				     <div class="col-md-4">
				     	    
				     	    <i class="fa fa-clock-o fa-4x" aria-hidden="true"></i>
				     	   <h1>1028 Hours of Works</h1>
				     </div>
				
				 <div class="col-md-4">
				     	
				     	 <i class="fa fa-user-plus fa-4x" aria-hidden="true"></i>
				     	 
				     	  <h1>94 Clients</h1>
				     	 
				     	 
				     </div>
				
				 <div class="col-md-4">
				     	
				     	<i class="fa fa-thumbs-up fa-4x" aria-hidden="true"></i>
				     	 <h1>571 Projects Delivered</h1>
				     </div>
				
				
			</div>
			
			
			
			
			
			
		</div>
		
		
		
		
		
		
		
		
		
		
		
		
		
		
	</section>   <! -- counter burda bitiyor-->
	
	
	<!--TEAM baslangic-->
	
	
	 <section id="team" class="text-center">
    	
    	
    	<div class="conteiner">
    		
    		
    		
    		 <div class="teamtop">
    		 	
    		 	<h1>TEAM</h1>
    		 	<p>Lorem ipsum dolor sit amet, numquam appareat liberavisse his ei, at eleifend praesent suavitate eam, nihil impetus apeirian ut est.<br> An nec sonet prompta. Quando assentior interesset duo ex, ea soluta 

patrioque nam. Audire admodum vivendum ei his, ut esse velit reformidans eum. Eos mazim choro ex, ancillae officiis salutandi pri an.
</p>


    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 </div>
    		
    		
    		<div class="teamContent wow fadeInRight">
    			
    			<div class="row">
    				
    				  <div class="col-md-4">
    				  	
    				  	<img src="img/team1.png"/>
    				  	
    				  	<h3>jim cary</h3>
    				  	<h4>Meneging Directoy</h4>
    				  	 <i class="fa fa-twitter fa-4x" aria-hidden="true"></i>
    				  	  <i class="fa fa-linkedin fa-4x" aria-hidden="true"></i>
    				  </div>
    				  
    				 <div class="col-md-4">
    				  	
    				  	<img src="img/team2.png"/>
    				  	<h3>Barat pitt</h3>
    				  		<h4>Lead Designer</h4>
    				  		<i class="fa fa-twitter fa-4x" aria-hidden="true"></i>
    				  	  <i class="fa fa-linkedin fa-4x" aria-hidden="true"></i>
    				  </div>
    				     
    				     
    				      <div class="col-md-4">
    				  	
    				  	<img src="img/team1.png"/>
    				  	<h3>jim cary</h3>
    				  	<h4>Lead Coder</h4>
    				  	<i class="fa fa-twitter fa-4x" aria-hidden="true"></i>
    				  	  <i class="fa fa-linkedin fa-4x" aria-hidden="true"></i>
    				  </div>
    				     
    				
    			</div>
    			
    			
    			
    			
    			
    			
    			
    			
    		</div>
    		
    		
    		
    	</div>
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    	
    </section>  <! -- TEAM burda bitiyor-->
	
	
	
	
	
	<! -- partners burdan basliyor-->
	
	<section id="partners" class="text-center">
		
		
		<div class="container">
			
				<div class="row wow fadeInDown">
			
			<div class="col-md-4">
				
				
				<img  src="img/partner1.png"/>
				
				
				
			</div>
			
			<div class="col-md-4">
				
				
				<img  src="img/partner2.png"/>
				
				
				
			</div>
			
			<div class="col-md-4">
				
				
				<img  src="img/partnr3.png"/>
				
				
				
			</div>
			
			
			
			
			
			
		</div>
		
			
			
			
		</div>
	
		
		
		
		
		
		
		
		
		
		
	</section> <!--parner burda son-->
	
	
	<!--contact burdaaa basliyor-->
	
	
	
	
	
	
	 <section id="contact" >
    	
    	
    	<div class="conteiner">
    		
    		
    		
    		 <div class="contacttop text-center">
    		 	
    		 	<h1>CONTACT</h1>
    		 	<p>Lorem ipsum dolor sit amet, numquam appareat liberavisse his ei, at eleifend praesent suavitate eam, nihil impetus apeirian ut est.<br> An nec sonet prompta. Quando assentior interesset duo ex, ea soluta 

patrioque nam. Audire admodum vivendum ei his, ut esse velit reformidans eum. Eos mazim choro ex, ancillae officiis salutandi pri an.
</p>


    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 	
    		 </div>
    		
    		
    		<div class="contactContent">
    			
    			<div class="row">
    				
    				 <div class="col-md-6">
    				 	
    				 	<form>
    				 		
    				 		  <fieldset>
    				 			
    				 		<input type="text" id="name" placeholder="NAME" />
    				 	       </fieldset>
    				 		
    				 		 <fieldset>
    				 			
    				 		<input type="text" id="e-mail" placeholder="E_MAIL" />
    				 	       </fieldset>
    				 		
    				 		 <fieldset>
    				 			
    				 		<textarea rows="5"cols="30" id="mesaj" placeholder="Mesaj..."></textarea>
    				 	       </fieldset>
    				 	       
    				 	        <fieldset>
    				 			
    				 		<input type="submit" value="SEND"  id="submit"/>
    				 	       </fieldset>
    				 	</form>
    				 	
    				 </div>
    				     
    				     
    				      <div class="col-md-6" id="yukari">
    				 	
    				 	             <h1>Our Adress</h1>
    				 	           <p>Gagarina 94-a ...</p>
    				 	           <h1>Call Us</h1>
    				 	           <p>+0380123456</p>
    				 	           <h1>E_mail-Adress</h1>
    				 	           <a mailto:info@erhan.com>erhanyener1492@gmail.com</a>
    				     </div>
    				     
    				
    			</div>
    			
    			
    			
    			
    			
    			
    			
    			
    		</div>
    		
    		
    		
    	</div> <!--contact burda sonnn-->
	




    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="js/bootstrap.min.js"></script>
    
    
                         <script src="js/wow.min.js"></script>
              <script>
              new WOW().init();
              </script>
    
    
    
    
    
    
    <script>
    	
    	
    	$(function() {
  $('a[href*="#"]:not([href="#"])').click(function() {
    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
      var target = $(this.hash);
      target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
      if (target.length) {
        $('html, body').animate({
          scrollTop: target.offset().top
        }, 1000);
        return false;
      }
    }
  });
});
    	
    	
    	
    </script>
    
    <script src="js/lightbox.js"></script>
    
  </body>
</html>