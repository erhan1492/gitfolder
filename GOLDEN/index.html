<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">  <!--kullanilan cihaza gore ayar-->
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>about me</title>

    <!-- Bootstrap -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/animate.css">
    <link rel="stylesheet" href="css/main.css">
<link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,400i" rel="stylesheet">

<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">

<link href="https://fonts.googleapis.com/css?family=Droid+Serif|Montserrat|Roboto" rel="stylesheet">


  </head>
  <body>


<section id="cover">
<div class="container">
	
	
		<div class="row">
		
		
<nav class="navbar navbar-toggleable-sm navbar-light fixed-top">
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
              </button>
                  <a class="navbar-brand" href="#">  </a>
                   <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
                      <ul class="navbar-nav">
                                  <li class="nav-item">
                                   <a class="nav-link" href="#cover">HOME<span class="sr-only">(current)</span></a>
                                   </li>
                        <li class="nav-item">
                                <a class="nav-link" href="#servis">SERVICES</a>
                              </li>
                                <li class="nav-item">
                                       <a class="nav-link" href="#portfolio">PORTFOLIO</a>
                                   </li>
                                 <li class="nav-item">
                                      <a class="nav-link" href="#about">ABOUT</a>
                                       </li>
                                       
                                       
                                         <li class="nav-item">
                                      <a class="nav-link" href="#contact">CONTACT</a>
                                       </li>
                     </ul> 
                 </div>
</nav>
	
	</div>  <!-- row bitis -->
	
	     <div class="row text-center">
	     	    <div class="col-sm-12">
	     	    	<h1>Welcome to our studio</h1>
	     	    	
	     	    </div>
	     	
	     	 <div class="col-sm-12">
	     	    	<h2>IT'S NICE TO MEET YOU</h2>
	     	    	
	     	    </div>
	     	
	     	 <div class="col-sm-12">
	     	   
	     	   <a href="#" ><button class="btn btn-primary btn-lg ">tell me more</button></a>
	     	    	
	     	    </div>
	     	
	     	
	     	
	     </div>
	
	
	
	
	
	
	
	
	
</div>

</section>




<section id="servis">
	
	<div class="container">
		
	<div class="servis-top wow wobble data-wow-duration="2s"">
		
		   <h3>SERVICES</h3>
		   <P>proin iacus purus cunsequat cure</P>
		
		
	</div>
		
		<div class="servis-content">
			
			      <div class="row text-center">
			      	  <div class="col-sm-4">
			      	  	
			      	  	<img class="wow shake" src="images/sepet.png"/>
			      	  	<h4>E-Commerce</h4>
			      	  	<p>ntesque augue sit eu dapibus nec hendrerit, suscipit pede in, nunc a pede, arcu odio                       </p>
			      	  	
			      	  </div>
			      	
			      	
			      	
			      	
			      	<div class="col-sm-4">
			      		
			      		<img class="wow shake" src="images/tele.png"/>
			      		      <h4>Responsive Web</h4>
			      	  	<p>ntesque augue sit eu dapibus nec hendrerit, suscipit pede in, nunc a pede, arcu odio                       </p>
			      		
			      	</div>
			      	<div class="col-sm-4">
			      		
			      		<img class="wow shake" src="images/kilid.png"/>
			      		
			      		<h4>Web Security</h4>
			      	  	<p>ntesque augue sit eu dapibus nec hendrerit, suscipit pede in, nunc a pede, arcu odio                       </p>
			      		
			      		
			      	</div>
			      	
			      	
			      </div>
			
		</div>
		
		
	</div>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</section>


























<section id="portfolio">
	
	
	<div class="container">
		
		
		<div class="portfolio-top text-center">
			
			
			<h3>PORTFOLIO</h3>
		   	<P>prain iaculis purus consequat sem cure</P>
			
			
		</div>
		
		<div class="portfolio-content">
			
			<div class="row">
				
				<div class="col-sm-4 wow flip">
					
					<div class="card">
					
						  <img  src="images/ebon.jpg" alt="produckt"/>
					<div class="card-block">
						<h4 class="card-title">Ebony &amp;Ivory</h4>
						<p class="card-subtitle text-muted">Branding</p>
						
					</div>
						
						
						
						
					
					
					</div>
					
				</div>
				
				<div class="col-sm-4 wow flip">
				
				    <div class="card">
				    	<img  src="images/ebon2.jpg"/>
				
				          <div class="card-block">
						<h4 class="card-title">Smart stationar</h4>
						<p class="card-subtitle text-muted">Print desing</p>
						
					</div>
				
				    	
				    	
				    	
				    	
				    </div>
				
				
				
				</div>
				
				<div class="col-sm-4 wow flip">
				
				
				    <div class="card">
				    	
				    	   <img  src="images/ebon3.jpg"/>
				
				           <div class="card-block">
						<h4 class="card-title">Clever poster</h4>
						<p class="card-subtitle text-muted">Print Desing</p>
						
					</div>
				
				    	
				    	
				    	
				    	
				    </div>
				
				
				
				</div>
				
				
			</div>
			
			<div class="row">
				
				<div class="col-sm-4 wow flip">
					
					<div class="card">
					         
					         
					         	
					         	   <img  src="images/roow2-1.jpg"/>
					
					       <div class="card-block">
						<h4 class="card-title">Vinly Record</h4>
						<p class="card-subtitle text-muted">Product Mock-UP</p>
						
					             </div>
					         	
					         	
					         	
					         	
					         </div>
					
					    
					
					
					
					
					
					</div>
					
					
				
				
				<div class="col-sm-4 wow flip">
				
				
				    <div class="card">
				    	
				    	<img  src="images/row2-2.jpg"/>
				
				            <div class="card-block">
						<h4 class="card-title">Treehouse Tamplate</h4>
						<p class="card-subtitle text-muted">Web Desing</p>
						
					</div>
				    	
				    	
				    	
				    	
				    	
				    	
				    	
				    </div>
				
				
				
				
				
				</div>
				
				<div class="col-sm-4 wow flip">
				
				    <div class="card">
				    	
				    	<img   src="images/row2-3.jpg"/>
				
				        <div class="card-block">
						<h4 class="card-title">Burned Logo</h4>
						<p class="card-subtitle text-muted">Branding</p>
						
					</div>
				    	
				    	
				    </div>
				
				
				
				</div>
				
				
			</div>
			
			
			
			
			
			
			
		</div>
		
		
		
		
		
		
		
		
		
	</div>
	
	
	
	
	
</section> <!--  --portfolyo-->





<section id="about">
	
	<div class="container">
		    <div class="about-top text-center wow rubberBand" data-wow-duration="5s">
		    	<h3>About us</h3>
		    	<p>proin icalus purus consequat sem cure.</p>
		    	
		    </div>
		<div class="about-content">
			<div class="row"> 
				<div class="col-sm-4 text-right">
					<h4>July 2014</h4>
					<h4>Our Humble Beginings</h4>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.</p>

				</div>
				<div class="col-sm-4 text-center wow swing"><img src="images/about1.jpg"/></div>
				<div class="col-sm-4"></div>
				
				
			</div>
			
			<div class="row">
				
				<div class="col-sm-4"></div>
				
				<div class="col-sm-4 text-center wow swing"><img src="images/about2.jpg"/></div>
				
				<div class="col-sm-4 text-left">
					<h4>January 2011</h4>
					<h4>Facing Startup Battles</h4>
                <p> Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean 
rhoncus posuere odio in tincidunt.</p>

				</div>
				
				
				
			</div>
	
			<div class="row">
				
				<div class="col-sm-4 text-right">
					<h4>December 2012</h4>

				<h4>Enter The Dark Days</h4>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni.</p>
				</div>
				
				<div class="col-sm-4 text-center wow swing">
					<img src="images/about3.jpg"/>
				</div>
				
				
				
				
				
			</div>
			
			
			
			<div class="row">
				<div class="col-sm-4"></div>
				
				<div class="col-sm-4 text-center wow swing"><img  src="images/about4.jpg"/></div>
				
				<div class="col-sm-4">
					<h4>february 2014</h4>


					<h4>Our Triumph</h4>
					<p> Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                       aenean rhoncus posuere odio in tincidunt. Proin 
                           iaculis purus consequat sem cure digni 
                                ssim. Donec porttitora entum suscipit aenean.</p>
				</div>
				
				
			</div>
			
			
			
			<div  class="row">
				
				
				<div class="col-sm-4 offset-sm-4">
					
					
					<div class="about-circle">
					
					
					<p class="yellow-text">OUR<br />
					  STORY<br />
					  CONTINNUES<br />...
					</p>
					
					
					</div>
				</div>
			</div>
			
		</div>
		
	</div>
	
	
	
	
	
	
	
	
	
	
	
</section> <!-- about-->







<section id="team">
	
	<div class="container">
		
		<div class="team-top text-center">
			
			<h3>TEAM</h3>
			<p>Proin iaculis purus consequat sem cure</p>
			
			<div class="team-content">
				
				
				    <div class="row">
				
				    <div class="col-sm-4 wow bounce" data-wow-duration="5s">
				    	
				    	    <img src="images/team1.png"/>
				    	        <h4>Kimberly thomson</h4>
				    	        <h5>Marketer</h5>
				    	    
				    	    <ul class="social-icons">
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-twitter" aria-hidden="true"></i>
				    	    	
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-facebook" aria-hidden="true"></i>
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-map-pin" aria-hidden="true"></i>
				    	    	
				    	    </ul>
				    	    
				    	    
				               </div>
				                 
				                 
				                 
				                 
				             <div class="col-sm-4 wow bounce" data-wow-duration="5s">
				             	
				             	
				             	
				             	
				             	  <img src="images/team1.png"/>
				             	
				             	<h4>Rico Massimo</h4>
				    	        <h5>Coder</h5>
				             	
				             	  <ul class="social-icons">
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-twitter" aria-hidden="true"></i>
				    	    	
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-facebook" aria-hidden="true"></i>
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-map-pin" aria-hidden="true"></i>
				    	    	
				    	    </ul>
				             	
				             	
				             </div>
				
				               <div class="col-sm-4 wow bounce" data-wow-duration="5s">
				               	  
				               	 
				    	        
				               	    <img src="images/team1.png"/>
				               	    
				               	     <h4>Uku Masom</h4>
				    	        <h5>Graphic Designer</h5>
				               	    
				               	  <ul class="social-icons">
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-twitter" aria-hidden="true"></i>
				    	    	
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-facebook" aria-hidden="true"></i>
				    	    	<li class="social-icon"></li>
				    	    	<i class="fa fa-map-pin" aria-hidden="true"></i>
				    	    	
				    	    </ul>
				               	
				               	
				               	
				               	
				               </div>
				
				
				
				
				
				
				
			             </div>
			
				
				
				
				<div class="row">
					
					<p>Proin iaculis purus consequat sem cure</p>
					
					
				</div>
				
				
				
				
			</div>
			
			
			
			
			
			
			
			
			
			
			
			
		</div>
		
		
		
		
		
		
	</div>
	
	
	
	
	
	
	
	
	
</section>





<section id="contact">
	
	
	<div class="container">
		<div class="contact-top text-center">
			<h4>CONTACT</h4>
			<P>proin icalus purus consequat sem cure</P>
		</div>
		
		<div class="contact-content">
			
			   <form class="golden-form" role="form" method="get" >
			   	
			     <div class="row">
			   	    	 
			   	    	 
  <div class="col-sm-6">
 <div class="form-element">
 	 <input type="text" class="golden-form-element" id="name" name="name" placeholder="your name*"/>
  
 	
 	
 </div>
			   	    	  	
			<div class="form-element">
 	 <input type="email" class="golden-form-element" id="email" name="email" placeholder="your password*"/>
  
 	
 	
 </div>   	    	  	
			
			<div class="form-element">
 <input type="text" class="golden-form-element" id="subject" name="subject" placeholder="subject*"/>
  
 	
 	
 </div>   	    	  	
			   	    	  	
	 </div>
			   	    	
			<div class="col-sm-6">
				
				<div class="form-element">
				
				<textarea class="golden-form-element" name="message" placeholder="messaj..."></textarea>
				
				
				
				</div>
				
				
				
				
			</div>	    	
			   	    	
			   	    	
			   	    </div>
			   	
			   	
			   	<div class="row justify-content-center">
			   		
			   		<div class="form-element">
			   			
			   			
			   			<input  class="btn btn-primary btn-lg" type="submit" id="submit" name="submit" value="send message"/>
			   			
			   			
			   		</div>
			   		
			   		
			   		
			   		
			   		
			   	</div>
			   	
			   	
			   	
			   	
			   	
			   	
			   	
			   	
			   </form>
			
			
			
			
			
			
			
			
			
			
			
			
			
		</div>
		
		
		
		
		
		
		
		
		
		
		
	</div>
	
	
	
	
	
	
	
	
	
</section>



<section id="footer">
	
	<div class="container">
		
		<div class="row">
			
			
			<div class="col-sm-4">
				
				
				<p>&copy;erhanyener1492@gmail.com</p>
			</div>
			
			<div class="col-sm-4">
				<ul class="company-social-icons">
					
					<li class="company-social-icon">
						<img src="images/facebook.png"/>
					</li>
					
					
					<li class="company-social-icon">
						<img src="images/twitter.png"/>
					</li>
					
					
					<li class="company-social-icon">
						<img src="images/prin.png"/>
					</li>
					
					
					<li class="company-social-icon">
						<img src="images/google.png"/>
					</li>
				</ul>
				
				
				
			</div>
			
			
			
			
			
			
			
			
			
		</div>
		
		
		
	</div>
	
	
	
	
	
	
	
</section>


































<script src="js/jquery-3.1.1.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"></script>

  <script src="js/bootstrap.min.js"></script>
  <script src="js/wow.js""></script>
              <script>
              
              </script>
  <script src="js/main.js"></script>


 <script>
    	
    	new WOW().init();
    
$(window).scroll(function(){
		var scrollPosition=$(window).scrollTop(),
		navbar=$('.fixed-top');
		
		if(scrollPosition>500){
			
			navbar.addClass('change-fixed-top');
		}
		else{
			navbar.removeClass('change-fixed-top');
		}
	}
	);
  $('a[href*="#"]:not([href="#"])').click(function() {
    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
      var target = $(this.hash);
      target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
      if (target.length) {
        $('html, body').stop()
        $('html, body').animate({
          scrollTop: target.offset().top
        }, 1000);
        return false;
      }
    }
  });

    	
    	
    	
    </script>
    









  </body>
</html>